hugegraph-pd/hg-pd-service/src/main/java/org/apache/hugegraph/pd/rest/API.java

```diff
@@ -31,7 +32,8 @@
 
 public class API {
 
-    public static final String VERSION = "3.6.3";
+    // Currently reading directly from the pom.
+    public static final String VERSION = VersionUtil.getPomVersion();
```

Comment:
or may consider using this way:
```xml
<build>
    <plugins>
        <plugin>
            <groupId>org.apache.maven.plugins</groupId>
            <artifactId>maven-jar-plugin</artifactId>
            <configuration>
                <archive>
                    <manifest>
                        <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
                    </manifest>
                </archive>
            </configuration>
        </plugin>
    </plugins>
</build>
```

```java
public static String getPomVersion() {
    return VersionUtil.class.getPackage().getImplementationVersion();
}
```

Update the implement in the common-module